docs/impact/index.md

```diff
@@ -1,48 +1,17 @@
-# Impact
+# BIDS Impact
 
 This document intends to convey how the BIDS Steering and Maintainers Group can
 support your proposed BIDS grant. In an effort to express the impact of BIDS in
 the brain imaging community, we have shared several of our traffic metrics.
 
```

Comment:
I think that this needs to better introduce the following sections of the page and (please chime in here @ericearl @Remi-Gau @christinerogers) an acknowledgements should be mentioned and linked? to somewhere further down on the page.